### PR TITLE
docs/flow: add screenshots for the UI in the debugging page

### PR DESCRIPTION
**Pull request by rfratto**

### CODE REVIEW

File: docs/sources/flow/monitoring/debugging.md [hunk 1]
```diff
@@ -21,13 +21,27 @@ server, which defaults to listening at `http://localhost:12345`.
...
+Clicking **View** navigates to the [Component detail page](#component-detail-page)
+for that component.
+
+Clicking the Grafana Agent logo navigates back to the home page.
+
+### Graph page
+
+![](../../../assets/ui-graph-page.png)
+
+The **Graph** page shows a graph view of all running components along with
```
[the remark this answers is not in the document]
Comment (rfratto) [reply docs/sources/flow/monitoring/debugging.md:38]: Yeah, running is maybe misleading. 

WDYT about:

> The **Graph* page shows a graph view of all components defined in the config file along with ...

[the remark this answers is not in the document]
Comment (mattdurham) [reply docs/sources/flow/monitoring/debugging.md:38]: Sounds good

**Review by mattdurham — APPROVED** [review]
LGTM

File: docs/sources/flow/monitoring/debugging.md [hunk 2]
```diff
@@ -21,13 +21,27 @@ server, which defaults to listening at `http://localhost:12345`.
...
-* The **Graph** page shows a graph view of all running components along with
-  their health. Clicking a component in the graph navigates to the detail page
-  for that component.
+![](../../../assets/ui-home-page.png)
+
+The home page shows a table of components defined in the config file along with
+their health. Clicking **View** navigates to the [Component detail page](#component-detail-page)
+for that component.
+
+Clicking the Grafana Agent logo navigates back to the home page.
```
Comment (karengermond) [inline docs/sources/flow/monitoring/debugging.md:32]:
```suggestion
Click the Grafana Agent logo to navigate back to the home page.
```

File: docs/sources/flow/monitoring/debugging.md [hunk 3]
```diff
@@ -21,13 +21,27 @@ server, which defaults to listening at `http://localhost:12345`.
...
 
-* The home page shows a table of running components along with their health.
-  Clicking **View** navigates to the detail page for that component.
-* The **Graph** page shows a graph view of all running components along with
-  their health. Clicking a component in the graph navigates to the detail page
-  for that component.
+![](../../../assets/ui-home-page.png)
+
+The home page shows a table of components defined in the config file along with
+their health. Clicking **View** navigates to the [Component detail page](#component-detail-page)
```
Comment (karengermond) [inline docs/sources/flow/monitoring/debugging.md:29]:
```suggestion
their health. 

Click **View** to navigate to the [Component detail page](#component-detail-page)
```

**Review by karengermond — APPROVED** [review]
LGTM!

**Pull request merged**